games-action/dave_gnukem/patches/dave_gnukem-1.0.patchset

```diff
@@ -0,0 +1,180 @@
+From 2eada0e4dccb0e5006adb367dbb1672cf79a43ac Mon Sep 17 00:00:00 2001
+From: Abdur-Raheem Idowu <abdurraheemidowu@gmail.com>
+Date: Fri, 27 Dec 2019 08:23:50 +0000
+Subject: Patched SDL includes
+
+
+diff --git a/src/djgraph.h b/src/djgraph.h
+index b2f1cc9..bb57015 100644
+--- a/src/djgraph.h
++++ b/src/djgraph.h
+@@ -9,7 +9,7 @@ Copyright (C) 1998-2018 David Joffe
+ #ifndef _DJGRAPH_H_
+ #define _DJGRAPH_H_
+ 
+-#include "SDL.h"
++#include <SDL/SDL.h>
```

Comment:
This is not correct. The proper way to use SDL is #include "SDL.h" and use sdl-config or pkg-config to get the correct compiler flags.

Reply:
Adding both pkg-config or sdl-config won't solve the problem, or you need to patch the Makefile for INCLUDEDIRS or use "make INCLUDEDIRS=* $jobArgs" use finddir for this, this would solve patching for SDL

Reply:
Could you please elaborate on that please @Begasus ? I didn't quite understand. Should I not use sdl-config/pkg-config?

Reply:
You can try, leave out the SDL in the patchset and check